enerdata/profiles/profile.py

```diff
@@ -347,6 +356,10 @@ class REProfileZone5(REProfile):
     climatic_zone = 5
 
 
+class REProfileHydraulic(REProfile):
+    pass
```

Comment:
I think this is not well implemented, if we are creating an empty class and the logic written in the base class with `if`and `else`

Reply:
ok @ecarreras , as you see define the get_range method in the subclass like this:

```python
class REProfileHydraulic(REProfile):
    @classmethod
    def get_range(cls, start, end):
        filename = path.join(
            path.dirname(path.realpath(__file__)), 'data/coefficients_HIDRO_RE.xlsx'
        )
        df = pd.read_excel(filename, sheet_name='hidraulic')
        key = df.keys()[0]
        cofs = []
        while start <= end:
            month = cls.translate_month[start.month]
            coff_value = float(df[df[key] == month]['Factor de funcionamiento'])
            coff = Coefficent(start, {'A': coff_value})
            cofs.append(coff)
            start += relativedelta(hours=1)
        return cofs
```